### ISSUE TITLE
Error on Part 4 - Section II - Step 6

### ISSUE THREAD

**Issue by CyberMario63** [Plailect/Guide#359]
On the FBI inject, step 6 says to put the new FBI_inject_with_banner.app to the root of your SD card. However, after doing this Decrypt9WIP fails to detect FBI_inject_with_banner.app.

If you put FBI_inject_with_banner.app into the file9 folder, where the hs.app is located, Decrypt9WIP will detect FBI_inject_with_banner.app.

**Comment by smiley2billion** [comment]
The working directory for Decrypt9 changed with a recent update ( https://github.com/d0k3/Decrypt9WIP/releases/tag/20160810 ).  I also had to move it into the /files9 folder for it to show up even though it says that the old directory should still work.  I also had to do this with hs.app when restoring it later on.  I'm assuming this will be addressed when the new version of the guide comes out with the updated CTRNAND procedure.
